riscv-sbi.md

```diff
@@ -55,20 +60,24 @@ environment. When executed in S-mode, it generates an environment-call-from-S-mo
 exception and performs no other operation. The required arguments are passed
 through registers *a0-a2* and the SBI call type is passed via register *a7*. Once
 the machine mode receives the trap, it identifies the type of SBI call from *a7*
-register and perform required operation. Individual SBI call signature and its
-purpose is described next.
+register and perform required operation. Any unsupported SBI call shoud return
+error i.e. ENOSYS to indicate the supervisor mode that it is not supported.
```

Comment:
It's -ENOSYS, not ENOSYS, right?

Reply:
Yes. It should be -ENOSYS.

Apologies for the typo.

Reply:
I know Linux defines ENOSYS, but I think the SBI spec should not rely on such an explicit dependency, but rather define its ENOSYS explicitly